micro_epsilon_scancontrol_driver/src/driver.cpp

```diff
@@ -421,23 +429,29 @@ namespace scancontrol_driver
         const std::shared_ptr<micro_epsilon_scancontrol_msgs::srv::SetResolution::Request> request,
         std::shared_ptr<micro_epsilon_scancontrol_msgs::srv::SetResolution::Response> response)
     {
-        if (response->return_code = StopProfileTransfer() < GENERAL_FUNCTION_OK){
-            // return true;
+        response->return_code = StopProfileTransfer();
+        if (response->return_code < GENERAL_FUNCTION_OK){
+            RCLCPP_WARN_STREAM(LOGGER, "Error while stopping transmission! Code: " << response->return_code);
+            return;
```

Comment:
I am not sure when the returns got commented out (they are not in the `main` branch`).

The return type is `bool`, so I would return `true` still. Note that this only means the service handled the request, not that it was successfull.

Reply:
That's interesting. It shows `void` as return type for me. I branched off of ros2-devel. 
In case this is still needs to be fixed, maybe we can make a separate PR/issue for it